tools/labs/templates/memory_mapping/kmmap/kmmap.c

```diff
@@ -140,12 +140,11 @@ static int my_seq_open(struct inode *inode, struct file *file)
 	return single_open(file, my_seq_show, NULL);
 }
 
-static const struct file_operations my_proc_file_ops = {
-	.owner   = THIS_MODULE,
-	.open    = my_seq_open,
-	.read    = seq_read,
-	.llseek  = seq_lseek,
-	.release = single_release,
+static const struct proc_ops my_proc_file_ops = {
```

Comment:
NIT: rename to "my_proc_ops"

Reply:
They are ops for the procfs file. `proc_ops`